src/api/app/models/event_subscription/generate_hash_for_subscriber.rb

```diff
@@ -0,0 +1,64 @@
+class EventSubscription
```

Comment:
@hennevogel continuing our IRC conversation.. I dont think a view helper is a good place for this code because they are modules not classes and are not a good place to store complex business logic of an application.